test/ApiClient.test.ts

```diff
@@ -230,7 +230,7 @@ describe('Publish authn', () => {
   })
 })
 
-describe('Subscribe', () => {
+describe.only('Subscribe', () => {
```

Comment:
You'll want to remove the only before merging

Reply:
😅  good catch